accepted/0000-payg-cloud-integration.md

```diff
@@ -0,0 +1,242 @@
+- Feature Name: payg_cloud
+- Start Date: 2021-08-11
+
+# Summary
+[summary]: #summary
+
+In the three major Public Clouds (AWS, GCP and Azure), SUSE:
+ - provides customized pay-as-you-go-specific product images (eg. SLES, SLES for SAP...)
+ - operates per-region RMT Servers mirroring repositories for products available as pay-as-you-go
+
+Pay-as-you-go instances all come with Cloud-specific "tokens" that authorizes the specific instance to download specific products from SUSE-operated RMT Servers. Pay-as-you-go instances are registered to the closest RMT Server at launch time (region/Server name is auto-determined).
+
+At update time, a zypper plugin forwards such "token" to the RMT Server via custom headers.
+To check repository accessibility RMT Servers have plugins which use Cloud-specific internal APIs to check whether the "token" is valid and which products it is entitled to.
+
+
+# Motivation
+[motivation]: #motivation
+
+Uyuni needs to load repository data from external sources to provide it to the registered machines.
+At the moment Uyuni can synchronize repositories from:
+  - the SCC CDN
+  - a plain RMT Server ("from-mirror setup", still requires connection to SCC)
+  - a plain directory exported from an RMT Server ("disconnected setup")
+  - a custom repository
+
+Presently, it is not possible to sync content from a SUSE-operated Cloud RMT Server, because:
+  - a Uyuni Server does not come with "tokens" (unlike pay-as-you-go instances, see above)
+  - Uyuni does not know how to pass them to RMT Servers
+  - Uyuni Server do not know which repositories are served via such RMT Servers
+  - Uyuni does not know for which repos/products the user is entitled for - which pay-as-you-go instances of which products he run
+
+Since Uyuni cannot contact the cloud RMT servers the only option for users is to contact SUSE and get one SCC account which allows to synchronize content from the SCC CDN.
```

Comment:
```suggestion
Since Uyuni cannot contact the Cloud RMT servers the only option for users is to contact SUSE and get one SCC subscription which allows them to synchronize content from the SCC CDN.
```

Note for your information: creating an SCC **account** per se is free, self-serviceable and considered not to be a problem. 
What hurts users in this scenario is the fact they have to contact SUSE to get a **subscription**, which requires calling a human (and a chain of human-based back office operations inside of SUSE).